templates/eightball/Inspector.tsx

```
@@ -0,0 +1,238 @@
+                setLoading(false)
+                return filePath
+            } catch (error) {
+                setLoading(false)
+                console.error('Error uploading image:', error)
+                return ''
+            }
+        },
+        [uploadImage, updateConfig]
+    )
+
+    const resetConfiguration = useCallback(() => {
```

Biome

[error] This hook specifies more dependencies than necessary: updateConfig

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)